### ISSUE TITLE
Why is the accuracy so low when I use model prediction?

### ISSUE THREAD

**Issue by Zzheng-6** [eriklindernoren/PyTorch-YOLOv3#675]
## What I'm trying to do
<Why did I train 100 epochs with 24 classes and 1515 images in the dataset detect.py Is the accuracy of the results seen in the output file so low?>

## What I've tried
<I executed the following command at the terminal：python detect.py --image_folder data/samples/    --checkpoint_model checkpoints/yolov3_ckpt_100.pth  --class_path data/custom/classes.names>

**Comment by Zzheng-6** [comment]
![image](https://user-images.githubusercontent.com/75076373/115731823-077fe980-a3ba-11eb-8b59-c3f145ed7912.png)
![image](https://user-images.githubusercontent.com/75076373/115731887-15356f00-a3ba-11eb-9552-cf4342e2cdd4.png)
![image](https://user-images.githubusercontent.com/75076373/115731938-1d8daa00-a3ba-11eb-9689-0405cf5282b0.png)

**Comment by Flova** [comment]
Could you provide more logs as stated in the issue template? This seems to be an training issue, therefore please provide the training params, tensorboard logs, training mAP and so on.

**Comment by Zzheng-6** [comment]
training params are as follows:--model_def config/yolov3-custom.cfg
--data_config config/custom.data
--pretrained_weights weights/darknet53.conv.74
Is it because I used darknet53.conv.74 weight training? Because my dataset may be different from coco dataset, my task is garbage classification.

When I wanted to check the training log on tensorboard, my pychar reported a mistake：
![image](https://user-images.githubusercontent.com/75076373/115886782-d966dc00-a483-11eb-8749-d48970e77ad0.png)

**Comment by Zzheng-6** [comment]
Thank you for reading my question. It's a real headache

**Comment by Zzheng-6** [comment]
My training log is empty：
![image](https://user-images.githubusercontent.com/75076373/115887320-6447d680-a484-11eb-848c-d0b245d86215.png)

**Comment by Flova** [comment]
Could you try to update to the newest version of this code?

**Comment by Zzheng-6** [comment]
OK, I'll download the latest code and see the results

**Comment by Flova** [comment]
Any updates regarding this problem?
